x/service/types/relay.go

```diff
@@ -5,17 +5,31 @@ package types
 // A value receiver is used to avoid overwriting any pre-existing signature
 func (req RelayRequest) GetSignableBytes() ([]byte, error) {
 	// set signature to nil
+	sig := req.Meta.Signature
 	req.Meta.Signature = nil
+
+	bz, err := req.Marshal()
+	if err == nil {
+		req.Meta.Signature = sig
```

Comment:
we dont need to do this as the reciever is not a pointer ie this takes a copy of the request and doesnt change it